packages/experimental/Button/src/ButtonTokens.ts

```diff
@@ -11,7 +11,6 @@ export const defaultButtonTokens: TokenSettings<ButtonTokens, Theme> = () =>
     medium: {
       padding: globalTokens.spacing.sNudge - globalTokens.stroke.width.thin,
       borderWidth: globalTokens.stroke.width.thin,
-      borderRadius: globalTokens.corner.radius.medium,
```

Comment:
You may need to look at our other buttons and make sure that they are not implementing their own borderRadius tokens. I think CompoundButton might have their own setting?

Reply:
Yes, I removed it from CompoundButton. FAB is also using it's own borderRadius but FAB uses ButtonCore props and tokens which don't contain sizes and shape props